### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by josschne**
Add travis automation, badges

### CODE REVIEW

**Comment by kofalt** [comment]
Wanna rebase?

**Comment by josschne** [comment]
@kofalt ping

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+sudo: false
+python:
+- '2.7'
+- '3.3'
```
Comment (kofalt) [inline .travis.yml]: Do we actually want to support python 3.x compatibility?

**Comment by kofalt** [comment]
LGTM pending consideration of python matrix.

File: .travis.yml (hunk 1 above)
Comment (gsfr) [inline .travis.yml]: We make no claim on python3 compatibility. In fact, webapp2 states that it is not python3 compatible, but I have never tried. Also, why 3.3 and not 3.5?

**Comment by gsfr** [comment]
@josschne In this repo, we typically rebase our feature branches rather than merge in master. Hence, @kofalt's request.

Also, let's finish our 'scripts'  naming conversation before this gets merged. Having `scripts` next to `bin` next to `run.sh` is a little too cluttered for me.

**Comment by josschne** [comment]
Created #96 to address @gsfr concerns. @kofalt @gsfr ping.

**Comment by gsfr** [comment]
:+1:

**Pull request merged**